moduleroot/.gitlab-ci.yml.erb

```diff
@@ -91,6 +91,22 @@ before_script:
   allow_failure: true
 <%         end -%>
 <%       end -%>
+<%       if options['except'] -%>
+<%         if options['except'].has_key?(check) -%>
+  except:
+<%           options['except'][check] && options['except'][check].each do |ref| -%>
+    - <%= ref %>
+<%           end -%>
+<%         end -%>
+<%       end -%>
+<%       if options['only'] -%>
```

Comment:
This could be merged into a single `if` instead of nested e.g.

```
<% unless options['only'].nil? || options['only']['check'].nil? -%>
    except:
<% options['only'][check].each do |ref| -%>
   - <%= ref %>
<%  end -%>
<%  end -%>
```